internal/controller/bindata/vsp/intel/99.vsp-pod.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
```

Comment:
Since there are 2 intel cards. Is it possible to make the folders more specific?
E.g.
Intel-IPU-E***
Intel-NetSec-Accelerator
Marvell-Octeon-10